ruddock/modules/users/templates/manage_member.html

```diff
@@ -0,0 +1,92 @@
+{% extends "layout.html" %}
```

Comment:
i think if you pass MEMBER_EDIT_FIELDS (see previous comments), you could populate the fields in this template with the pre-existing values. otherwise the user will probably think they have to fill out every field